third_party/openthread/platforms/efr32/BUILD.gn

```diff
@@ -28,15 +29,16 @@ config("openthread_efr32_config") {
   include_dirs = [
     "${chip_root}/examples/platform/efr32",
     "${chip_root}/examples/platform/efr32/${efr32_family}/${efr32_board}",
-    "${openthread_root}/examples/platforms/efr32/src",
-    "${openthread_root}/examples/platforms/efr32/${efr32_family}",
+    "${openthread_efr32_root}/src/src",
+    "${openthread_efr32_root}/src/${efr32_family}",
   ]
 }
 
 source_set("openthread_core_config_efr32") {
   sources = [
-    "${openthread_root}/examples/platforms/efr32/src/openthread-core-efr32-config-check.h",
-    "${openthread_root}/examples/platforms/efr32/src/openthread-core-efr32-config.h",
+    "${openthread_efr32_root}/src/src/openthread-core-efr32-config-check.h",
+    "${openthread_efr32_root}/src/src/openthread-core-efr32-config.h",
+    "${openthread_root}/src/cli/cli_config.h",
```

Comment:
@mspang, could you please help us removing this ugly hack?

We had to do it in order to bypass this error:

ERROR at //third_party/connectedhomeip/third_party/openthread/repo/examples/apps/cli/cli_uart.cpp:37:11: Include not allowed.
#include "cli/cli_config.h"
          ^---------------
It is not in any dependency of
  //third_party/connectedhomeip/third_party/openthread/platforms:libopenthread-cli-uart
The include file is in the target(s):
  //third_party/connectedhomeip/third_party/openthread/repo/src/cli:libopenthread-cli-ftd
  //third_party/connectedhomeip/third_party/openthread/repo/src/cli:libopenthread-cli-mtd